showyourwork.yml

```diff
@@ -1,18 +1,27 @@
-# User configuration; see the docs for details!
+version: 0.3.0.dev0
 verbose: true
+rule_fallback: skip
```

Comment:
I got rid of this setting; won't exist in v0.3.0

Reply:
One of the error messages told me to add it :D

Reply:
Just get rid of it when you upgrade to the actual release. I found a much better way of dealing with missing input files -- you can probably find it if you poke around the tip of the dev branch. Snakemake keeps track of files with no "producers" but catches the exception if the file exists on disk. Showyourwork now re-raises the exception when that happens. User can override this behavior -- see config docs. I'll put a new prerelease on pypi for you tomorrow